features/functests/sctp/sctp.go

```diff
@@ -102,9 +191,24 @@ func loadMC() *mcfgv1.MachineConfig {
 	return mc
 }
 
+func featureLatencySensitiveOn() bool {
```

Comment:
this could be done using the type safe client. I was using it before the feature gate was enabled by default.
You can check it here https://github.com/fedepaol/sctp_test/commit/ee11f006b88fc797714b228fb406b3f5c95fb573#diff-d9cedae8764d88c1b05a71fb306ee070 (on the part that was being removed because there was no need to set the feature gate anymore)